examples/helloworld/deployment.yaml

```diff
@@ -14,7 +14,7 @@ spec:
     spec:
       containers:
       - name: helloworld
-        image: helloworld-image
+        image: "{{helloworld-image}}"
```

Comment:
The style of substitution through `resolver` is brittle because the path structure has to be known to the resolver in order to work.

The style of substitution, `"{{helloworld-image}}"`, through `templating/main.go` will work everywhere and no prior knowledge of the yaml structure will need to be baked into the code.

Reply:
I'd rather keep canonical image reference in the example
```
image: //helloworld:image
```